### PR TITLE
Prepare test & prod deployment

### PR DESCRIPTION
**Pull request by hwbllmnn**
@KaiVolland Please review.

### CODE REVIEW

File: setEnvironment.sh
Comment (KaiVolland) [inline setEnvironment.sh:1]: Why was this file removed?

Comment (hwbllmnn) [reply setEnvironment.sh:1]: This repo is only used for test/prod deployments, so this is not necessary any more.

**Review by KaiVolland — APPROVED** [review]
